sphinx/writers/latex.py

```diff
@@ -399,6 +401,15 @@ def __init__(self, document, builder):
         if self.elements['polyglossia']:
             self.elements['babel'] = ''  # disable babel forcely
             self.elements['multilingual'] = self.elements['polyglossia']
+            self.elements['classoptions'] += ',' + self.babel.get_language()
+            if builder.config.language:
```

Comment:
The `\setmainlanguage` is called only if users set up `language` variable. Is this intended?
If the function should be always called, this line should be removed.